cabal-install/Distribution/Client/SetupWrapper.hs

```diff
@@ -439,12 +471,14 @@ externalSetupMethod path verbosity options _ args = do
       searchpath <- programSearchPathAsPATHVar
                     (map ProgramSearchPathDir (useExtraPathEnv options) ++
                       getProgramSearchPath (useProgramDb options))
-      env        <- getEffectiveEnvironment [("PATH", Just searchpath)
+      env'        <- getEffectiveEnvironment [("PATH", Just searchpath)
                                             ,("HASKELL_DIST_DIR", Just (useDistPref options))]
 
-      process <- runProcess path' args
-                  (useWorkingDir options) env Nothing
+      -- TODO use createProcess
```

Comment:
Can we add the same TODO at the definition of runProcess', that really we should just switch all uses over, or be more deliberate in our creation of a `createProcess` wrapper.